cmd/kubeadm/app/cmd/reset.go

```diff
@@ -68,15 +69,21 @@ func NewCmdReset(in io.Reader, out io.Writer) *cobra.Command {
 				kubeadmutil.CheckErr(err)
 			}
 
+			cfg, err := configutil.FetchInitConfigurationFromCluster(client, os.Stdout, "reset", false)
+			if err != nil {
+				cfg = nil
```

Comment:
cfg is already nil, see:
https://github.com/kubernetes/kubernetes/blob/739af85d7a63dec41e07913c14c3fe26695804bb/cmd/kubeadm/app/util/config/cluster.go#L42

Reply:
ok, actually the PR looks good already. we seem to check `cfg != nil` in all places where needed.
my mistake!
^ please only remove this `cfg = nil` line.

Reply:
OK.

i will check the code that using `cfg` inside `Run` functions and add check before using `cfg`.

Reply:
no, no the PR is good already.
just remove `cfg = nil` here